.gitignore

```diff
@@ -4,3 +4,5 @@ coverage/
 package-lock.json
 .env
 .DS_Store
+cypress/screenshots/
+cypress/videos/
```

Comment:
That sounds like a good idea -- I'll add `"video": false` to `cypress.json`. I would still like to keep `cypress/videos/` in `.gitignore` just in case a contributor enables recording video by hand, so that video files don't end up getting accidentally committed.